kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/Food2Door/OrderProcessor.java

```diff
@@ -0,0 +1,12 @@
+package com.kodilla.good.patterns.challenges.Food2Door;
+
+public class OrderProcessor {
+
+    public void orderProcess(Order order) {
+        order.getSupplier().process(order);
```

Comment:
to jest b. ubogie rozwiazanie. Dobrze byloby dac tutaj jakas minimalna mechanike